packages/@aws-cdk/aws-redshift/README.md

```diff
@@ -15,4 +15,54 @@
 ---
 <!--END STABILITY BANNER-->
 
+### Starting a Redshift Cluster Database
+
+To set up a Redshift cluster, define a `RedshiftCluster`. It will be launched in a VPC.
+You can specify a VPC, otherwise one will be created. The nodes are always launched in private Subnets and are encrypted by default.
+
+``` typescript
+const cluster = new RedshiftCluster(this, 'Redshift', {
```

Comment:
We follow a different convention:

1. Import the package with the service name.
2. Do not prefix the class names with the name of the service.

So, this should be:

```ts
import redshift = require('@aws-cdk/aws-redshift');

const cluster = new redshift.Cluster(this, 'Redshift', {
    masterUser: {
      masterUsername: 'admin',
    },
    vpc
});
```